### ISSUE TITLE
import blacklist from 'the-big-username-blacklist' does not work.

### ISSUE THREAD

**Issue by joonhocho** [marteinn/The-Big-Username-Blacklist-JS#2]
I am using babel / webpack from react-redux-universal-hot-example.
It seems this package does not support `import` syntax.
I tried `require` and it works.

**Comment by marteinn** [comment]
Hi @joonhocho and thanks for the issue report!

I put together a quick test and things seems to work for me?

``` javascript
import {validate} from "the-big-username-blacklist";

console.log(validate("user"));
console.log(validate("martin"));
console.log(validate("about"));
console.log(validate("git"));
console.log(validate("system"));
console.log(validate("admin"));

// babel index.js --out-file index-test.js && node index-test.js
```

How does your import statement look like?

**Comment by joonhocho** [comment]
I see what was wrong. Here's what my code looked like:

```
import blacklist from "the-big-username-blacklist";
blacklist.validate('user'); // blacklist is undefined.
```

You must use `import {}` syntax to get it working.

**Comment by joonhocho** [comment]
Maybe change it to `export default` ?

**Comment by marteinn** [comment]
@joonhocho Already done and it is included in the latest 1.2.0 release :+1:  Thanks for the feedback!
